lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -289,8 +289,10 @@ defmodule ExUnit.Case do
             "\"use ExUnit.Case\" in the current module"
     end
 
-    tags = tags ++ Module.get_attribute(mod, :tag) ++ moduletag
-    tags = tags |> normalize_tags |> Map.merge(%{line: env.line, file: env.file})
+    tags =
+      tags ++ Module.get_attribute(mod, :tag) ++ moduletag
+      |> normalize_tags
```

Comment:
sorry I don't  quite get what you mean eric,
do you have an example to look at?